eng/AfterSigning.targets

```diff
@@ -1,9 +1,7 @@
 <Project>
   <PropertyGroup Condition="'$(GenerateChecksums)' == 'true'">
-    <!-- The one use of ArtifactsDir in Publish.proj adds an additional slash, confusing itself. -->
```

Comment:
Hmm, missed whatever update removed use of `$(ArtifactsDir)` in Publish.proj. But, I still see an extra slash at https://github.com/dotnet/arcade/blob/b63ea25b8dc50bb3dfcef128d415254bd5ca4dc8/src/Microsoft.DotNet.Arcade.Sdk/tools/Publish.proj#L267 and am not sure manifests will generate correctly w/ this change and that.

Reply:
WIll verify. That's `$(ArtifactsLogDir)` though, and https://github.com/dotnet/arcade/blob/b63ea25b8dc50bb3dfcef128d415254bd5ca4dc8/src/Microsoft.DotNet.Arcade.Sdk/tools/Publish.proj#L35 suggests that it does

> I'm not a fan of the extra changes in this PR given we've been fine w/ https://github.com/dotnet/aspnetcore/blob/master/eng/Publishing.props#L8 and https://github.com/dotnet/aspnetcore/blob/master/eng/AfterSigning.targets#L4 Can we leave that out of this PR, reducing it to the required changes in eng/Signing.props❔
> 
> And, could you explain why eng/Publishing.props needs to mention signing items❔

I doesn't work without those changes. It will result in a different paths being seen Signing.props based on context. The problem here is that the values coming into Signing.props for `$(ArtifactsDir)` vary based on whether https://github.com/dotnet/aspnetcore/pull/26033/files#diff-5e710afb117de5939bdf18e05b6aacd3L4 has been run. This causes globbing to miss files. If I don't fix this, then I either have to duplicate all the globbing here: https://github.com/dotnet/aspnetcore/pull/26033/files#diff-3f0380cc81441e712c5eba1d7b77e649R17-R42 to handle different paths in both cases, or I have to put in another hack to add or remove a trailing slash.

I'll verify that the same files get published, signed, etc. That's pretty trivial to do, but I really want to fix this. I haven't seen this in any other repo, and the subtlety of the alteration in the AfterSigning.targets took **two days** to find. It was very difficult to find out why the same globbing strings worked during signing but not after.

Reply:
Yes, this makes sense. Thanks for the detailed explanation and your verification the manifests didn't change.

> the subtlety of the alteration in the AfterSigning.targets took **two days** to find.

I included that fix in eng/Publishing.props when I first created the file as part of our conversion to use the stages pipeline. My apologies for your frustration.

Reply:
> I included that fix in eng/Publishing.props when I first created the file as part of our conversion to use the stages pipeline. My apologies for your frustration.

No worries. I was very happy when I found it :D